webpack/assets/javascripts/react_app/components/ReportTemplatesTable/index.js

```diff
@@ -0,0 +1,12 @@
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import ReportTemplatesTable from './ReportTemplatesTable';
+import reducer from './ReportTemplatesTableReducer';
+import * as actions from './ReportTemplatesTableActions';
+
+const mapStateToProps = state => state.report_templates_table;
+const mapDispatchToProps = dispatch => bindActionCreators(actions, dispatch);
+
+export const reducers = { report_templates_table: reducer };
```

Comment:
I know the models table is snake case as well, could we rather use camel case instead?